src/components/Wiki/WikiPage/InsightComponents/CurrencyConverter.tsx

```diff
@@ -114,32 +120,48 @@ const CurrencyConverter = ({ token, tokenSymbol }: CurrencyConverterProps) => {
         <Box p={2} mt={1}>
           <HStack
             flexDirection={isTokenLeft ? 'row' : 'row-reverse'}
-            p={2}
-            borderRadius={4}
+            px={3}
+            borderRadius={6}
             justify="space-between"
             bgColor="wikiCardItemBg"
+            position="relative"
+            overflow="hidden"
           >
+            <Box
+              pos="absolute"
+              top={0}
+              right={0}
+              w="50%"
+              bgColor="gray.50"
+              _dark={{
+                bgColor: 'gray.600',
+              }}
+              h="100%"
+              zIndex={1}
+            />
             <CurrencyBox
               token={token}
               tokenSymbol={tokenSymbol}
               value={fromCurrency}
               setValue={e => updateValues(e, true)}
             />
             <IconButton
-              bgColor="dimColor"
+              bgColor="gray.100"
               borderWidth="1px"
-              borderColor="borderColor"
+              color="gray.500"
               aria-label="convert"
               cursor="pointer"
-              _hover={{ bgColor: '#0000003a' }}
-              _focus={{ bgColor: '#0000003a' }}
-              _active={{ bgColor: '#0000003a' }}
+              _dark={{ bgColor: 'gray.800' }}
+              _hover={{ filter: 'brightness(95%)' }}
+              _focus={{ filter: 'brightness(95%)' }}
+              _active={{ filter: 'brightness(95%)' }}
```

Comment:
To help reduce repetition
```suggestion
              sx={{
                '&:hover, &:focus, &:active': {
                  filter: 'brightness(95%)',
                },
              }}
```

Reply:
That's awesome ! thanks